docs/plugins.md

```diff
@@ -1,4 +1,4 @@
----
+﻿---
```

Comment:
Please what is this change for? It might break the page metadata?

Reply:
@ghybs thanks for the review.  That particular change you highlighted was not intentional (I don't have any client side source control installed and it may have been an artifact when I did file upload).  Thanks for any correction.